bundles/NewsletterBundle/public/js/document/newsletters/sendingPanel.js

```diff
@@ -79,8 +79,14 @@ pimcore.document.newsletters.sendingPanel = Class.create({
             });
 
             var sourceAdapterList = [];
-            var adapterNames = Object.keys(pimcore.document.newsletters.addressSourceAdapters);
+            var adapterNames = Object.keys(pimcore.bundle.newsletter.document.newsletters.addressSourceAdapters);
             for(var i = 0; i < adapterNames.length; i++) {
+                //skip adapters that require a namespace that is not available
+                let requiredNamespace = pimcore.bundle.newsletter.document.newsletters.addressSourceAdapters[adapterNames[i]].prototype.requiredNamespace;
+                if (requiredNamespace && typeof pimcore.bundle[requiredNamespace]['startup'] === 'undefined') {
```

Comment:
LGTM, ready to be merged but one final small change for more flexibility

```suggestion
                if (requiredNamespace && typeof pimcore.bundle[requiredNamespace] === 'undefined') {
```

1. there could be a bundle that call the initialization differently than `startup` or doesn't have at all.
2.  if i set a `requiredNamespace` wrongly eg. `customzzreportz`, it would result in 
![image](https://user-images.githubusercontent.com/6014195/231243856-230d105b-a40a-4c02-8021-90686db3bf3c.png)
While without startup, it would skip that without errors, treat it as if not installed/available because it would be undefined.

Reply:
1. that was my first attempt but there are bundles which registers the same namespace like CMF registers a class `pimcore.bundle.customreports.custom.definition.termSegmentBuilder` so it would result in wrong enable.

2. It is not user provided value so it makes sense to fail in case of typos.

Reply:
ahhh ok, then let's merge it as is, thank you for the details behind that